src/operator/custom/custom-inl.h

```diff
@@ -129,6 +129,9 @@ class CustomOperator {
           ctx.run_ctx.ctx, vars, vars2, FnProperty::kNormal, 0,
           "CustomOperator");
     });
+    // increase num_threads if there is not enough threads to execute custom operator
+    if (q_.size() > num_free_threads)
```

Comment:
I'm not sure if this is a good behavior. If the number of ops to execute is too many, it keeps creating threads and the number of threads may keep going up unbounded.

Reply:
MXNET_CUSTOM_OP_NUM_THREADS is the upbound number of threads.